### DISCUSSION TITLE
Overlay/scribble strip

### DISCUSSION THREAD

**Discussion by mhuss99** [clevelandmusicco/HothouseExamples#13] [Ideas]
It would be really cool if Cleveland Music (or anybody) could create a "scribble strip" plastic overlay for the Hothouse to easily label the controls without sticky tape. I'd buy some. :)

**Comment by clevelandmusicco** [comment]
Hey now, that's a pretty cool idea. I don't know if you're a synth person, but overlays are common in that world. For example, here's a Moog Mother-32 without, and then with, an overlay.

![image](https://github.com/user-attachments/assets/82a38be1-aa1c-4742-80d7-2897c73e222c)  

In the pic above, the overlay is affixed to the synth face via the case screws, but we don't easily have that option with the Hothouse (you'd have to remove all the knobs and dress nuts).

Something loose-fitting that is fairly disposable might also be useful?

![image](https://github.com/user-attachments/assets/2fc79e30-00c6-45d4-a51e-c51493a4a770)  

That's the same synth, but with a blank card stock overlay. For the Hothouse, none of the controls would be labelled and one could just write them in as needed. I could easily see those being cost-effective.

Maybe I could share some templates for use on a CNC laser cutter. One could easily engrave a custom faceplate (or a totally blank one for scribbling) with 1/16" one or [two-color acrylic](https://www.inventables.com/products/dark-green-on-white-laserable-acrylic-sheet?variant=41037038092345). Actually, some companies always use a faceplate on their pedals to lower manufacturing costs.

The super deluxe option: if you write an effect that lives on your Hothouse forever, you could always design a custom enclosure and order from Tayda. Here's the [drill template](https://drill.taydakits.com/box-designs/new?public_key=Q2luT1FRTVZRM3Bab01YbXd3eituUT09Cg==)

**Comment by mhuss99** [comment]
Since the main point of the Daisy/Hothouse is experimentation, I think an inexpensive disposable version would be most useful. I'm more of an amp guy, and I've had a number of custom faceplates made for amp builds, but I think the loose-fitting use-and-toss model would make the most sense for this.

**Comment by clevelandmusicco** [comment]
> Since the main point of the Daisy/Hothouse is experimentation, I think an inexpensive disposable version would be most useful.

While you're 100% correct re: the Hothouse, in the future, you just might see some commercially available Cleveland Music Co. pedals that have the same form factor and control layout as the Hothouse, but with entirely different artwork. And under the hood, they just might be a Hothouse with a pre-programmed Daisy Seed (commercial use being the device's _other_ use case). Just maybe :wink: :shrug:

I'm adding a disposable overlay to the backlog. Thanks for the idea!

**Comment by dlwhite66** [comment]
I'm just getting started and had this same idea.. Meris offered these (1st round was bad, 2nd pass was decent) for their Enzo and Polymoon pedals.  It would be great to have a template for these and somewhere to order custom ones because doing it at home on a laser printer - cutting the holes would be problematic. 
![image](https://github.com/user-attachments/assets/36c560c4-b1b0-41d9-a324-a6737b74a4be)

**Reply by dlwhite66** [reply]
might be worth seeing if these guys will offer low quantities...hmmm.   https://www.label-aid.com/products/graphic-overlays/digital-graphic-overlays

**Reply by clevelandmusicco** [reply]
The demand has been low, so I don't think I'd ever bulk order. When I originally thought about this, I figured something like a [Cricut machine](https://cricut.com/en-us/cutting-machines/cricut-joy/cricut-joy/cricut-joy/2007991.html) would fit the bill pretty well. I haven't researched how portable the design files used in their software might be, however. More research required ...

I've seen these Cricut devices at various Hacker/Maker Spaces and they're fairly inexpensive. Someone could even cut a bunch of these blank overlays and sell them on Reverb or summat. Just thinking and typing ...

**Comment by clevelandmusicco** [comment]
Update on this idea: given the DIY/learning/experimentation spirit of the Hothouse, I'll leave this to the community. (Read: I don't make much money with the Hothouse and don't particularly care to; that was never the point. 😉) 

There are all manner of cheap and easy solutions here. I recently started doing what we do in recording studios with large consoles and patchbays: use [console tape](https://www.rosebrand.com/product871/Artists-Console-Tape.aspx). It's low-tack and leaves no residue on the Hothouse. Quick, easy, and super cheap. I use [this 1/2" size tape](https://www.amazon.com/ProTapes-Artist-Flatback-Printable-Console/dp/B00DVB0T10/).
